db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_04_13_180915) do
+ActiveRecord::Schema.define(version: 2022_06_09_001128) do
```

Comment:
I'm a bit concerned by this schema change, but it does reflect a problem we were seeing in production cc @jeremyf should this be included?

Reply:
I believe that should be included.  But agree with focusing on the task at hand.    However, without the above parser fix, we have a problem.

This code, with the schema change was deployed to production; so I believe we're already seeing this "in play".